student/pom.xml

```diff
@@ -330,6 +336,10 @@
             <artifactId>jsp-api</artifactId>
             <scope>provided</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.assertj</groupId>
+			<artifactId>assertj-core</artifactId>
```

Comment:
Fixed all the assertj test scopes.  Nice catch.  I was using them to assert the values were the same in the remote and legacy objects.